jobs/p_rabbitmq_alerts/templates/p_rabbitmq.alerts

```diff
@@ -33,3 +33,28 @@ ALERT PRabbitMQServiceBrokerDown
     summary = "Rabbitmq for PCF Service Broker instance `{{$labels.environment}}/{{$labels.bosh_deployment}}/{{$labels.bosh_job_name}}/{{$labels.bosh_job_ip}}` is down",
     description = "The RabbitMQ Service Broker instance at `{{$labels.environment}}/{{$labels.bosh_deployment}}/{{$labels.bosh_job_name}}/{{$labels.bosh_job_ip}}` has been down for the last <%= p('p_rabbitmq_alerts.service_broker_down.evaluation_time') %>",
   }
+
+<% if_p("p_rabbitmq_alerts.queue_depth.warning") do |threshold|  -%>
+ALERT RabbitMQQueueDepth
+  IF avg(firehose_value_metric_p_rabbitmq_p_rabbitmq_rabbitmq_messages_depth) by(environment) >= <%= threshold %>
```

Comment:
@geofffranks Curious here, we tend to add a `FOR` to the alters to avoid false alarms. For example, if the `RabbitMQQueueDepth` exceeds the threshold here, it will raise immediately an alert, wouldn't it be better to raise the alert if it exceeds the threshold during a certain amount of time (ie 5mins)?